### ISSUE TITLE
Issue with importing into state

### ISSUE THREAD

**Issue by jeffmacdonald** [terraform-providers/terraform-provider-github#397]
### Terraform Version

0.12.24

Github provider 2.4.1

### Affected Resource(s)

github_repository

### Terraform Configuration Files
```hcl
provider "github" {
  # We get the token from the environment variable GITHUB_TOKEN
  organization = "foo"
  version      = "2.4.1"
}

provider "aws" {
  region  = "us-east-2"
  version = "2.54"
}

resource "github_repository" "this" {
  name               = "jeffmacdonald"
  description        = "Jeff Testing"
  private            = true
  has_issues         = false
  has_projects       = false
  allow_merge_commit = true
  allow_squash_merge = true
  allow_rebase_merge = true
  has_wiki           = false
  archived           = false
}
```

### Debug Output
Please provide a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

### Panic Output

None.

### Expected Behavior
So, basically I want to import github repositories into my state. When I run `terraform import github_repository.this jeffmacdonald` , I'm able verify the repo is now in the state. 

I would expect that when I run `tf plan` that it might tell me it has to modify certain resources.

### Actual Behavior

What actually happens, is that `tf plan` tells me there is `1 to add, 0 to change, 0 to destroy`. If I then run `tf apply` the GitHub api complains that a repo with that name already exists.

### Steps to Reproduce
1. Add a new github_repo to your tf file
2. Import it from github into your state
3. attempt to apply.

### Important Factoids

I'm using Terraform Cloud.

### References
This FEELS related to https://github.com/terraform-providers/terraform-provider-github/issues/388

**Comment by jeffmacdonald** [comment]
This is fixed as of 2.6.0!
